src/shared/api.js

```diff
@@ -49,11 +49,18 @@ const getCoordinatesData = async (address) => {
     .catch((error) => console.error("Error geocoding:", error));
 };
 
+const getHead = () => {
+  return fetch(`http://localhost:3000/`, {
+    method: "HEAD",
+  }).catch((error) => console.error("Error fetching venues:", error));
```

Comment:
This error message is missleading. You are not fetching venues. You are checking if server is running.